test/crypto/rfc-scheme.test.ts

```diff
@@ -0,0 +1,5 @@
+describe("rfc compliant scheme", () => {
+    it("should pass test vectors from go codebase", () => {
+
```

Comment:
Feels like this one is missing the tests, no?

Reply:
oops, have added it